Mathlib/RepresentationTheory/GroupCohomology/Resolution.lean

```diff
@@ -375,11 +374,16 @@ theorem diagonalHomEquiv_symm_apply (f : (Fin n → G) → A) (x : Fin (n + 1) 
     Category.comp_id, Action.comp_hom, MonoidalClosed.linearHomEquivComm_symm_hom]
   -- Porting note: This is a sure sign that coercions for morphisms in `ModuleCat`
   -- are still not set up properly.
+/- Note: The next four lines was
   -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
   erw [ModuleCat.coe_comp]
-  simp only [ModuleCat.coe_comp, Function.comp_apply]
+  simp only [ModuleCat.coe_comp, Function.comp_apply] -/
+  change (TensorProduct.uncurry k (of (Representation.ofMulAction k G G)) (of 1) A
+    (leftRegularHom ((ihom (of 1)).obj A) (Finsupp.llift A k k (Fin n → G) f)).hom
+    (((Representation.ofMulAction k G G).repOfTprodIso 1).hom.hom
+      ((diagonalSucc k G n).hom.hom (fun₀ | x => (1 : k))))) = _
   -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
-  erw [diagonalSucc_hom_single]
+  erw [diagonalSucc_hom_single x (1 : k)]
```

Comment:
This one looks a bit worrying

Reply:
It's not necessary now, but this will be faster.

Reply:
I think best to revert anything that isn't direct fallout of this change. Any other cleanups can go in a separate PR if you think they're still a good idea